src/MatrixClientPeg.js

```diff
@@ -128,6 +129,9 @@ class MatrixClientPeg {
             opts.lazyLoadMembers = true;
         }
 
+        const color_scheme = SettingsStore.getValue("roomColor");
```

Comment:
Initializing the UI feels a bit out of touch with a method that is initializing the js-sdk (just a suggestion as we already do the matrix action creators in here). How about putting this in `MatrixChat::_onWillStartClient` or in the "sync" event handler somewhere there?

Reply:
Good call. Have moved it.